packages/realm/bindgen/src/templates/browser.ts

```diff
@@ -909,34 +912,25 @@ class BrowserCppDecls extends CppDecls {
         void ${c.jsName}_deleter(emscripten::val pointer) {
           // FIXME reenable when investigating currently causing 'Uncaught RuntimeError: memory access out of bounds' 
           // delete reinterpret_cast<${c.cppName}*>(pointer.as<std::uintptr_t>());
+          static_cast<void>(pointer);
         }        
       `);
     });
 
-    // export free functions via embind
+    // export method functions via embind
     out(`\nEMSCRIPTEN_BINDINGS(realm_c_api) {`);
-    this.free_funcs
-      .filter((free_func: CppFunc) => {
-        // FIXME:  manually exclude some function signatrue causing issue wiht embind (ex: call to deleted constructor of 'realm::ThreadSafeReference')
```

Comment:
Are these fixed now?

Reply:
They are excluded at an earlier level - only the functions in `this.methodFunctions` are declared inside the `EMSCRIPTEN_BINDINGS` block.